<launch>

<!-- Default folder with maps-->
<arg name="folder" default="$(env HOME)"/>

<!-- Distance Measurement Node -->
<node pkg="stroll_bearnav" type="odometry_monitor_node" name="odometry_monitor">
	<!--remap from="odom" to="/naoqi_driver/odom"/-->
</node>

<!-- feature extraction -->
  <node pkg="stroll_bearnav" type="feature_extraction_node" name="feature_extraction">
	  <!--remap from="image" to="/pepper_robot/naoqi_driver/camera/bottom/image_raw"/-->
	  <!--remap from="image" to="/naoqi_driver/camera/front/image_raw"/-->
	  <remap from="image" to="/stereo/left/image_raw"/>
  </node>

<!-- mapping node -->
<node pkg="stroll_bearnav" type="mapper_node" name="mapper" output="screen">
  <param name="folder" value="$(arg folder)" />
  <param name="angularSpeed" value="1.0" />
</node>

</launch>
